USERGUIDE.md

```diff
@@ -39,6 +41,7 @@ Information on how to get the key name and private key is documented in [the REA
 ```Python
 import uuid
 from target365_sdk import ApiClient
+from target365_sdk.api_client import OutMessage
```

Comment:
According to the tests the correct import for OutMessage is "from ..models.out_message import OutMessage"

Reply:
This will not work in client code, because ..models.out is a library-internal reference. For clients target365_sdk will exist in a pip store, not in a relative path.

Reply:
Good point! 👍